1-development/bahamasclient/app/models/client.rb

```diff
@@ -0,0 +1,6 @@
+class Client < ApplicationRecord
+  has_and_belongs_to_many :invoices
```

Comment:
In our current model, a client has many invoices and an invoice belongs to a single client.
What would be the approach based on the current model?